src/test/resources/zoldicatalina/suite.testing.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
You either try to use your groups/classes from teorie/TestNg package or you remove this file. Thanks!